docs/data/material/getting-started/templates/marketing-page/components/FAQ.tsx

```diff
@@ -6,15 +6,18 @@ import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
 import Link from '@mui/material/Link';
 import Typography from '@mui/material/Typography';
-
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
 export default function FAQ() {
-  const [expanded, setExpanded] = React.useState<string | false>(false);
```

Comment:
Oh god, I hate this so much any time I land on a website that does this. I think https://tailwindui.com/all-access does this right.